pkgs/applications/networking/instant-messengers/nheko/default.nix

```diff
@@ -1,8 +1,10 @@
 { lib, stdenv, fetchFromGitHub
 , cmake, cmark, lmdb, qt5, qtmacextras, mtxclient
-, boost, spdlog, olm, pkgconfig
+, boost, spdlog, olm, pkgconfig, nlohmann_json
 }:
 
+# These hashes and revisions are based on those from here:
+# https://github.com/Nheko-Reborn/nheko/blob/v0.6.4/deps/CMakeLists.txt#L52
```

Comment:
Seems to be not relevant anymore, is it?

Reply:
So it seems that the comment is there to tell the next person looking at the package that the hashes in `rev` are imported from that `CMakeLists` file...

I'd rather put this note in the commit message, which is (IMO, tbh) the first place to look at when curious about such a thing... not in the file itself at some other place...

Maybe someone else has a different opinion though...

Reply:
Although I'm a beginner Nix user, I did understand what these lines meant - I figured these are dependencies pulled manually by the Nix build system. Therefor, I figured that I may need to update these as well according to the new version's requirement. My first attempt was to search in nheko's source tree for any mention of what specific version does it pull. At first I thought these will be git submodules versions but then I've reached that `CMakeLists.txt` file and found these revisions' hashes.

My opinion is that when someone will want to update Nheko to 7.0.0, hopefully he will see this comment which importantly has a v0.6.4 URL, and will check if the revisions have had an update in 7.0.0.

Additionally, not every contributor's `git blame` skills are good enough that he will easily find the original commit message (not mine) that added these dependencies which have hopefully (I haven't checked that myself) mentioned where did the revisions came from.